llama-index-integrations/vector_stores/llama-index-vector-stores-vespa/llama_index/schema/vespa/vespa_node.py

```diff
@@ -0,0 +1,5 @@
+from llama_index.core.schema import TextNode
+
+
+class VespaNode(TextNode):
+    vespa_fields: dict
```

Comment:
Did we need a new object class for this? I wonder if this could just be a metadata field?